specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -0,0 +1,641 @@
+            "description": "A pdf document or image (jpg,png) file to analyze.",
+            "required": true,
+            "type": "file"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AnalyzeResult"
+            }
+          },
+          "default": {
+            "description": "Response entity accompanying non-successful responses containing additional details about the error.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Model from id": {
+            "$ref": "./examples/Analyze.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "TrainRequest": {
+      "description": "Contract to initiate a train request.",
+      "required": [
+        "source"
+      ],
+      "type": "object",
+      "properties": {
+        "source": {
+          "description": "Get or set source path.",
+          "maxLength": 2048,
+          "minLength": 0,
+          "type": "string"
+        }
+      }
+    },
+    "TrainResult": {
+      "description": "Response of the Train API call.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Identifier of the model.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "trainingDocuments": {
+          "description": "List of documents used to train the model and the\r\ntrain operation error reported by each.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormDocumentReport"
+          }
+        },
+        "errors": {
+          "description": "Errors returned during the training operation.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FormOperationError"
+          }
+        }
+      }
+    },
+    "KeysResult": {
+      "description": "Result of an operation to get\r\nthe keys extracted by a model.",
+      "type": "object",
+      "properties": {
+        "clusters": {
+          "description": "Object mapping ClusterIds to Key lists.",
+          "type": "object",
+          "additionalProperties": {
+            "uniqueItems": false,
+            "type": "array",
+            "items": { "type": "string" }
+          }
+        }
+      }
+    },
+    "ModelResult": {
+      "description": "Result of a model status query operation.",
+      "type": "object",
+      "properties": {
+        "modelId": {
+          "description": "Get or set model identifier.",
+          "type": "string",
+          "format": "uuid",
+          "x-nullable": false
+        },
+        "status": {
+          "description": "Get or set the status of model.",
+          "enum": [ "created", "ready", "invalid" ],
```

Comment:
@ramparab any reason you do not want this to be x-ms-enum and take advantage of modelAsString?

Reply:
This swagger contains two 'Status' enums with different values which is causing issues.